server/src-rsr/initialise.sql

```diff
@@ -657,7 +657,7 @@ CREATE VIEW hdb_catalog.hdb_computed_field_function AS
   FROM hdb_catalog.hdb_computed_field
 );
 
-CREATE OR REPLACE FUNCTION hdb_catalog.check_violation(msg text) RETURNS bool AS 
+CREATE OR REPLACE FUNCTION hdb_catalog.check_violation(msg text) RETURNS bool AS
```

Comment:
maybe we should revert this to avoid diffs of completely unrelated parts of the code